lib/melhor-envio/new-label.js

```diff
@@ -31,6 +30,12 @@ module.exports = (order, appConfig, merchantData) => {
     postal_code: zip || '',
     note: shippingLines.to.near_to || ''
   }
+  
+  if (order.buyers && Array.isArray(order.buyers) && order.buyers[0] && order.buyers[0].registry_type === 'j') {
```

Comment:
```suggestion
  if (order.buyers[0].registry_type === 'j') {
```

Não precisa tantos checks aqui porque neste ponto já bateu `order.buyers[0].main_email` (sem verificar, não que esteja certo :neutral_face: ) por exemplo, então se não `order.buyers && Array.isArray(order.buyers) && order.buyers[0]` já teria dado um fatal antes..